stacks/spica/src/bucket/pages/bucket-add/bucket-add.component.html

```diff
@@ -229,12 +231,14 @@ <h4 mat-subheader><mat-icon>text_fields</mat-icon> Properties</h4>
               <mat-icon *ngIf="propertyKv.value.readOnly" matTooltip="This property is read-only."
                 >format_clear</mat-icon
               >
-            </ng-template>
-
+            </div>
             <span
+              class="input-placer-area"
```

Comment:
You can use `.input-placer-input` class as well.

Reply:
It is added by input placer by default

Reply:
doesn't exists  .input-placer-input  class. 
![image](https://user-images.githubusercontent.com/52313317/61998713-c6f81c80-b0bc-11e9-8b26-ef3103410f2e.png)

Reply:
Okay then we can pass this